OConnor/resources/views/purchase_item.html

```diff
@@ -53,7 +53,7 @@
 			successCallback: newData,
 			failureCallback: onFailure,
 			filterArray: [
-			              LABKEY.Filter.create('key', checkedRows, LABKEY.Filter.Types.EQUALS)
+			              LABKEY.Filter.create('key', checkedRows[0], LABKEY.Filter.Types.EQUALS)
```

Comment:
Should the button support more than one row?  If so, you could change the filter to an EQUALS_ONE_OF. 

Reply:
The code a little higher up explicitly checks that only one row is selected.